airflow/api_connexion/openapi/v1.yaml

```diff
@@ -412,25 +412,50 @@ paths:
         - $ref: '#/components/parameters/PageOffset'
         - $ref: '#/components/parameters/OrderBy'
         - $ref: '#/components/parameters/FilterTags'
-        - name: only_active
+        - $ref: '#/components/parameters/OnlyActive'
+        - name: dag_id_pattern
           in: query
           schema:
-            type: boolean
-            default: true
+            type: string
           required: false
           description: |
-            Only return active DAGs.
+            If set, only return DAGs with dag_ids matching this pattern.
```

Comment:
We should remove dag_id_pattern on /dags endpoint and make it a separate PR then add tests for it. Currently, there's no test for it and it will be good to make this change in a separate PR

Reply:
I hadn't added the `dag_id_pattern`, this was added by @SamWheating in [this](https://github.com/apache/airflow/pull/18924/files) PR.
I can see that the field is existing in the [main](https://github.com/apache/airflow/blob/a4d514bd7f7777e5bf03e9fa8e567f1c8774a74d/airflow/api_connexion/openapi/v1.yaml#L417) branch. It looks like the git diff is [misleading](https://github.com/apache/airflow/blame/bc8d9ba119ea7112a25d67aa70a6ea6e5069052b/airflow/api_connexion/openapi/v1.yaml#L412).